gqueries/general/converter_groups/chps_per_plant_type/chp_coal.gql

```diff
@@ -0,0 +1,6 @@
+- query =
+    FILTER(
+      Q(electricity_producers_coal),
+      "steam_hot_water_output_conversion > 0"
```

Comment:
We typically specify both digits, so `0.0` instead of `0`. This ensures that the number is read as a float, instead of as an integer.